rtt/typekit/BoolTypeInfo.hpp

```diff
@@ -49,10 +49,10 @@ namespace RTT
         /**
          * Write boolean as 'true' or 'false'.
          */
-        struct BoolTypeInfo: public TemplateTypeInfo<bool,true>
+        struct BoolTypeInfo: public StdTypeInfo<bool>
         {
             BoolTypeInfo() :
-                TemplateTypeInfo<bool,true> ("bool")
+                StdTypeInfo<bool> ("bool")
             {
```

Comment:
@meyerj Can we consider these changes ABI compatible? (I guess no-one directly uses BoolTypeInfo.hpp)